packages/upset/src/components/ElementView/ElementSidebar.tsx

```diff
@@ -98,17 +100,16 @@ export const ElementSidebar = ({ yOffset }: { yOffset: number }) => {
     <>
       <Drawer
         sx={{
-          width: hide ? 0 : fullWidth ? '100%' : drawerWidth,
+          width: hideElementSidebar ? 0 : fullWidth ? '100%' : drawerWidth,
           flexShrink: 0,
           '& .MuiDrawer-paper': {
             padding: '1em',
-            paddingTop: `${yOffset}px`,
-            width: hide ? 0 : fullWidth ? '100%' : drawerWidth,
-            visibility: hide ? 'hidden' : 'initial',
+            paddingTop: `${yOffset + 5}px`,
```

Comment:
Why add 5 here? Can this not just be included in the `yOffset` variable?

Reply:
Addressed in commit 152aa75